src/superplaceholder.js

```diff
@@ -99,17 +112,20 @@
 	};
 
 	var superplaceholder = function (params) {
-		if (!isPlaceHolderSupported) { return; }
+		if (!isPlaceHolderSupported) {
```

Comment:
IMO, flow breaking statements (like return, continue etc) can be left on the same line.